### ISSUE TITLE
Custom/Extended Fields Error

### ISSUE THREAD

**Issue by lowpez** [anchorcms/anchor-cms#1125]
Hello guys,
Thaks for any advice ;)

### Summary
Can't Add. Can't  Edit/Update any post.

I'm getting this error in a  fresh install, after a custom image field was created for be used in posts.
This field is used for add a simple featured image.

Related to this thread https://github.com/anchorcms/anchor-cms/issues/1070 .
I already tried the recommended solution by @sashahohloma without good results.

### OUTPUT message
```
Missing argument 2 for Extend::resizeImage(), called in C:\UwAmp\www\blobcl\anchor\models\extend.php on line 172 and defined
The error has been logged in /anchor/errors.log

Uncaught Exception

Missing argument 2 for Extend::resizeImage(), called in C:\UwAmp\www\blobcl\anchor\models\extend.php on line 172 and defined
Origin

anchor\models\extend.php on line 188
Trace

#0 C:\UwAmp\www\blobcl\anchor\models\extend.php(188): System\Error::native(2, 'Missing argumen...', 'C:\\UwAmp\\www\\bl...', 188, Array)
#1 C:\UwAmp\www\blobcl\anchor\models\extend.php(172): Extend::resizeImage('C:\\UwAmp\\www\\bl...')
#2 [internal function]: Extend::process_image(Object(stdClass), '2')
#3 C:\UwAmp\www\blobcl\anchor\models\extend.php(244): call_user_func_array(Array, Array)
#4 C:\UwAmp\www\blobcl\anchor\routes\posts.php(267): Extend::process('post', '2')
#5 [internal function]: {closure}()
#6 C:\UwAmp\www\blobcl\system\route.php(152): call_user_func_array(Object(Closure), Array)
#7 C:\UwAmp\www\blobcl\system\router.php(152): System\Route->run()
#8 C:\UwAmp\www\blobcl\system\start.php(46): System\Router->dispatch()
#9 C:\UwAmp\www\blobcl\index.php(35): require('C:\\UwAmp\\www\\bl...')
#10 {main}
```

### Context details (if applicable)

- Anchor version: 0.12.1
- Server setup: Apache, MySQL 5.7, and PHP 5.6
- URL : Running Local

**Comment by daviddarnes** [comment]
Hmm, `resizeImage()` coming up in the error. Are you trying to upload an image to this custom field?

Also, can you fill in the above information correctly please? It really helps if we have the PHP version number and a bit more context

**Comment by lowpez** [comment]
Hola @daviddarnes Thanks a lot for your time & patience.
Post updated.

**Comment by daviddarnes** [comment]
Thanks! Do you have any other custom fields running on this install? Is it just images that break?

**Comment by lowpez** [comment]
@daviddarnes just this one and yes, i tried a clean post, without any Custom/Extended field,
everything worked fine untill this change.

Cheers.

**Comment by daviddarnes** [comment]
I'm thinking it might be an issue with what you have installed locally, considering the error is coming from `resizeImage`. Any thoughts @TheBrenny?

**Comment by lowpez** [comment]
Alright then @daviddarnes!
I'll try a new install (online) in a few. I'll keep you inform ;) 

Greetings.

**Comment by TheBrenny** [comment]
I think we've got a mismatch in versions here, because in the current master, [line 172](https://github.com/anchorcms/anchor-cms/blob/7a8ba70d8c9fb97827eaabd907d919eaa74c28b5/anchor/models/extend.php#L172) is just a single curly brace, so it can't be calling anything... Taking that into account though, I've fished back to when line 172 *DID* call something. Back on [Feb 24th](https://github.com/anchorcms/anchor-cms/commit/608d2fd0d565fd970e912e64ca92bedcf7f5dc14#diff-d84b039efa691ea6153d4fd7769012e6R172), the `resizeImage` was introduced, and Craig probably forgot to overload the function, or forgot to add the parameter in. It just happens, but you were unfortunate enough to be victim to this accident. Nonetheless, it was been fixed in [commit 8d5fe7f](https://github.com/anchorcms/anchor-cms/commit/8d5fe7f7dd5eab454d619a8f79c96c4d2bd2c56c#diff-d84b039efa691ea6153d4fd7769012e6) thanks to a few other bug reports. All you should really need to do is upgrade anchor, and the problem is solved. I would normally suggest that you overwrite just the single file, but because there have been a few commits that change a lot of files, I recommend you upgrade all of Anchor.

## tl;dr
Backup your databases and files, then [upgrade Anchor](http://docs.anchorcms.com/getting-started/upgrading/#upgrading-from-08-and-above).

**Comment by lowpez** [comment]
@TheBrenny @daviddarnes 
Thanks a lot guys!. All working now.

PS: Guys i have an Update for the  Spanish Traslation. If you're intersted, please let me know ;)

**Comment by TheBrenny** [comment]
@lowpez, probably best to send a PR to the [Anchor Translations](https://github.com/anchorcms/anchor-translations) repo with the updated Spanish translations. :smiley:
